dist/components.css

```diff
@@ -1,4 +1,9 @@
 /*!
+* Techne v2.0.0
```

Comment:
should the version number be v2.0.0-alpha.1 to match what we'll have in the package.json?